cvat-sdk/cvat_sdk/auto_annotation/driver.py

```diff
@@ -16,13 +16,39 @@
 from cvat_sdk.core.progress import NullProgressReporter, ProgressReporter
 from cvat_sdk.datasets.task_dataset import TaskDataset
 
+from ..attributes import attribute_value_validator
 from .exceptions import BadFunctionError
 from .interface import DetectionFunction, DetectionFunctionContext, DetectionFunctionSpec
 
 
+@attrs.frozen
+class _AttributeNameMapping:
+    name: str
+
+
 @attrs.frozen
 class _SublabelNameMapping:
     name: str
+    attributes: Optional[Mapping[str, _AttributeNameMapping]] = attrs.field(
+        kw_only=True, default=None
+    )
+
+    def map_attribute(self, name: str) -> _AttributeNameMapping:
+        if self.attributes is None:
+            return _AttributeNameMapping(name)
+
+        return self.attributes.get(name)
```

Comment:
Probably, this call can return `None`, which contradicts the `map_attribute()` return type annotation.

Reply:
Thanks, fixed.